### DISCUSSION TITLE
wxWidgets on macOS Big Sur - alternative approach

### DISCUSSION THREAD

**Discussion by vashpan** [PCemOnMac/PCemV17macOS#7] [General]
Hello

Thanks for the fork as PCem indeed would benefit from better macOS support, although it would be the best to integrate those changes to mainline app - I hope this is possible.

One thing about macOS BigSur and wxWidgets: there's no need to build it manually. There's a custom Homebrew-tap for that so it could be installed easily:

`brew install cdalvaro/tap/wxmac` 

(https://github.com/cdalvaro/homebrew-tap)

(previous `wxmac` package should be uninstalled: `brew uninstall wxmac`)

**Comment by kyr0** [comment]
Ah nice, if you'd like to come up with a PR, pls go ahead :)
